init-odkx-sync-endpoint.py

```diff
@@ -125,6 +174,34 @@ def replaceInFile(file_path: str , pattern: str, subst: str):
     remove(file_path)
     move(abs_path, file_path)
 
+def setup_certificate(env: Dict[str, str]):
```

Comment:
These function name is a little misleading. The skip_setup_certificate function also does setup a certificate, it's just done via cerbot instead of manually as done here. I would call this one setup_manual_certificate and the other one setup_certbot_certificate or something along those lines.